lib/main_common.pm

```diff
@@ -630,6 +630,16 @@ sub we_is_applicable {
       && get_var('MIGRATION_REMOVE_ADDONS', '') !~ /we/;
 }
 
+sub libreoffice_is_applicable {
+    # for opensuse libreoffice package has ExclusiveArch:  aarch64 %{ix86} x86_64
+    # do not know for SLE (so assume built for all)
+    return 1 if (!is_opensuse);
+    return (check_var('ARCH', 'x86_64')
```

Comment:
So you want to exclude ppc64le?

Reply:
I voluntarily choose here the same logic as in spec that has ExclusiveArch and not ExcludeArch
https://build.opensuse.org/package/view_file/openSUSE:Factory:PowerPC/libreoffice/libreoffice.spec?expand=1

Reply:
@foursixnine do you really WANT to change the logic ?

Reply:
@michelmno I think the return 1 was kind of a mistake by me :P, in any case :) I have no arguments against your logic, specially if you intented to have it this way.